c/array-patterns/array4_pattern.c

```diff
@@ -62,6 +62,8 @@ int main()
 
 		array1[index1] = num * (num * count) ;
 		array2[index2] = num * count ;
+		if(count == 200)
+			count = 0 ;
```

Comment:
Is 200 a magic number?

Reply:
Just used it as a simple reset condition. One can put any other number for checking. Based on analysis on past year's benchmarks, 200 is the max. number of unrolling one needs to do on any reachsafety benchmarks in Svcomp to get property violation. The intention is to check the robustness of ml or prediction algorithms that the tools are deploying for algorithm selections. Although, it depends whether these numbers are taken as parameters to deploy falsification and correctness checking algorithms by a tool.

Reply:
A comment in the source would be nice, if still doable.